app/controllers/price_groups_controller.rb

```diff
@@ -31,20 +31,17 @@ def show
 
   # GET /facilities/:facility_id/price_groups/:id/users
   def users
-    unless SettingsHelper.feature_on?(:user_based_price_groups) && @price_group_ability.can?(:read, UserPriceGroupMember)
-      raise ActiveRecord::RecordNotFound
-    end
-
-    @user_members = @price_group.user_price_group_members
-                                .includes(:user)
-                                .joins(:user)
-                                .merge(User.sort_last_first)
-    @user_members = paginate(@user_members)
-    @tab = :users
+    price_group_members
```

Comment:
Good idea to create a method for shared logic.  I'd vote for putting the feature flag and authorization check into a `before_action`, then you can keep the new method more focused on setting `@user_members`:
```
def users
  @user_members = price_groups_members
  @tab = :users

  render action: "show"
end
```